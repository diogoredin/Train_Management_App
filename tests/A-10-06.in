4
4
Ã‰vora
0
0
