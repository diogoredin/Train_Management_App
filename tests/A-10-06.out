Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Consulta de serviços
1 - Mostrar todos os serviços
2 - Mostrar serviço com um dado número
3 - Mostrar serviços com início em estação
4 - Mostrar serviços com término em estação
0 - Sair
Escolha uma opção: Nome da estação: Serviço #590 @ 12.20
07:02 Lisboa - Oriente
07:10 Lisboa - Entrecampos
07:14 Lisboa - Sete Rios
07:26 Pragal
07:48 Pinhal Novo
07:57 Poceirão
08:01 Fernando Pó
08:06 Pegoes
08:10 São João das Craveiras
08:18 Vendas Novas
08:32 Casa Branca
08:42 Évora
Serviço #570 @ 21.20
10:02 Lisboa - Oriente
10:10 Lisboa - Entrecampos
10:14 Lisboa - Sete Rios
10:26 Pragal
10:48 Pinhal Novo
11:19 Grândola
11:33 Ermidas-Sado
11:55 Funcheira
12:24 Santa Clara-Sabóia
12:52 Messines-Alte
13:02 Tunes
13:07 Albufeira - Ferreiras
13:19 Loulé
13:30 Évora
Serviço #598 @ 12.20
19:02 Lisboa - Oriente
19:10 Lisboa - Entrecampos
19:14 Lisboa - Sete Rios
19:26 Pragal
19:48 Pinhal Novo
20:10 Vendas Novas
20:24 Casa Branca
20:35 Évora
Serviço #596 @ 12.20
17:02 Lisboa - Oriente
17:10 Lisboa - Entrecampos
17:14 Lisboa - Sete Rios
17:26 Pragal
17:48 Pinhal Novo
18:10 Vendas Novas
18:24 Casa Branca
20:39 Évora
Serviço #592 @ 12.20
09:02 Lisboa - Oriente
09:10 Lisboa - Entrecampos
09:14 Lisboa - Sete Rios
09:26 Pragal
09:48 Pinhal Novo
10:10 Vendas Novas
10:24 Casa Branca
22:35 Évora
Consulta de serviços
1 - Mostrar todos os serviços
2 - Mostrar serviço com um dado número
3 - Mostrar serviços com início em estação
4 - Mostrar serviços com término em estação
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 